### ISSUE TITLE
Load Experimental Extension No Cancel

### ISSUE THREAD

**Issue by 2jour** [LLK/scratchx#64]
OS: mac osx 10.10.5
Browser: Chrome 46

1. Go to https://scratchx.org/#scratch
2. Drag a few blocks
3. Click Load Experimental Extensions

[TODO Insert photo in Github]

You will notice there is no cancel button. If you refresh screen data is lost.

Not obvious to click Open button with blank github url

Marking it high because this can cause data loss.

**Comment by 2jour** [comment]
related to https://github.com/LLK/scratchx/issues/63

**Comment by kaschm** [comment]
@2jour I don't think this is prio-high. The way out of this dialogue is to click the black, though we can talk about design changes if this is not intuitive. Can you please confirm that clicking on the black works for you?

**Comment by 2jour** [comment]
works but not intuitive.  I will remove priority label.

**Comment by 2jour** [comment]
https://github.com/LLK/scratchx/issues/49
